event-processing/event-processor/tests/unit/app.test.ts

```diff
@@ -363,11 +363,11 @@ describe('Unit test for app handler', function () {
         await handler(sqsEvent);
 
         expect(consoleMock).toHaveBeenCalledTimes(5);
-        expect(consoleMock).toHaveBeenNthCalledWith(1, '[ERROR] VALIDATION ERROR\n{"requireFieldErrors":[{"sqsResourceName":"arn:aws:sqs:us-west-2:123456789012:SQSQueue","eventId":"66258f3e-82fc-4f61-9ba0-62424e1f06b4","eventName":"","timestamp":"1609462861","requiredField":"event_name","message":"event_name is a required field."}]}')
-        expect(consoleMock).toHaveBeenNthCalledWith(2, 'Topic ARN: SOME-SNS-TOPIC');
-        expect(consoleMock).toHaveBeenNthCalledWith(3, 'MessageID is 1');
-        expect(consoleMock).toHaveBeenNthCalledWith(4, 'Topic ARN: SOME-SNS-TOPIC');
-        expect(consoleMock).toHaveBeenNthCalledWith(5, 'MessageID is 2');
+        expect(consoleMock).toHaveBeenNthCalledWith(1, 'Topic ARN: SOME-SNS-TOPIC');
+        expect(consoleMock).toHaveBeenNthCalledWith(2, 'MessageID is 1');
+        expect(consoleMock).toHaveBeenNthCalledWith(3, 'Topic ARN: SOME-SNS-TOPIC');
+        expect(consoleMock).toHaveBeenNthCalledWith(4, 'MessageID is 2');
+        expect(consoleMock).toHaveBeenNthCalledWith(5, '[ERROR] VALIDATION ERROR\n{"requireFieldError":{"sqsResourceName":"arn:aws:sqs:us-west-2:123456789012:SQSQueue","eventId":"66258f3e-82fc-4f61-9ba0-62424e1f06b4","eventName":"","timestamp":"1609462861","requiredField":"event_name","message":"event_name is a required field."}}');
         expect(sns.publish).toHaveBeenCalledWith(
             {
                 Message: expectedResultOne,
```

Comment:
Are lines 377-381 successful and correct? I would have expected that `sns.publish` would not have been called for the invalid event, so what am I missing?

Reply:
It includes 2 valid messages and one invalid message. So the calls to SNS publish are valid ones.

Reply:
Ah, I didn't see we were sending the same message twice. Why are we doing that? And this is a tiny nibly detail so it's fine to leave: if it's the same message, why do we have expectedResultOne and expectedResultTwo and not use just one?

Reply:
That's just me missing the obvious. I will make that change to tidy it up.

Reply:
I've pushed a change that should tidy that up.